search/cities_boundaries_table.hpp

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include "indexer/city_boundary.hpp"
+
+#include <cstdint>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
+class Index;
+
+namespace search
+{
+class CitiesBoundariesTable
+{
+public:
+  class Boundaries
+  {
+  public:
+    Boundaries() = default;
+
+    Boundaries(std::vector<indexer::CityBoundary> const & boundaries, double eps)
+      : m_boundaries(boundaries), m_eps(eps)
+    {
+    }
+
+    Boundaries(std::vector<indexer::CityBoundary> && boundaries, double eps)
+      : m_boundaries(std::move(boundaries)), m_eps(eps)
```

Comment:
Opt. How about changing order of params at these two constructors to place changeable param at the end?

Reply:
No, boundaries is more significant than eps. Eps is not significant :)

Reply:
Ok